app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max: string;
+  value: string;
+  children?: React.ReactNode;
+};
+
+type SliderStates = {
+  value: string;
+  max: string;
+};
+
+export class Slider extends React.Component<SliderProps, SliderStates> {
+  state: SliderStates = {
+    value: this.props.value,
+    max: this.props.max,
+  };
+
+  handleChange = (event: any): void => {
+    this.setState({ value: event.target.value });
+  };
+
+  timeFormat = (time: string): string => {
+    const convertedTime = parseInt(time);
+    const minute = ~~(convertedTime / 60);
+    const seconds = convertedTime % 60;
+
+    const res =
+      (convertedTime < 60 ? "00" : minute < 10 ? "0" + minute : minute) +
```

Comment:
Imagine someone trying to understand this code and change afterwards - I bet they will spend ton of time understanding this code despite the fact that it's doing some simple things.

As Robert Martin said in his famous Clean Code book - [the only valid measurement of code is number of WTFs per minute](https://i.redd.it/btd5aebzr5h01.jpg)

This code on lines 30-41 made me say WTF 10 times in 10 seconds. 

**Why is that?**

- Why you convert from string to int to convert back to string?
- Why do you use ternary operators so obsessively? This code looks like a very good example why we should [mostly avoid ternary operators](https://pressupinc.com/blog/2015/03/ternary-operators-considered-harmful/). This one is so unreadable, if you will return to this code few months from now - I bet my ass you will struggle
- Why do you use a verb `format` in the name of the function and use `convert` verb inside the function - while both of them describe basically the same thing?
- Why your variable minute is singular, while seconds is plural? Can't minute become minutes?
- What is `res`? Even if I know that it's short for `result`, it doesn't cost you anything to write the full version of it

**And one more thing**

Right now your slider can be controlled only using mouse, but in fact it will both be controlled by song progress and by mouse, now you only support updating by dragging.
[Also as I stated in the issue](https://github.com/crococoders/tchaikovsky/issues/23)
> • Scrubbing line must appear on hover at the edge of the line

It means that when you start dragging the line back and forth it needs to have vertical bar in the end. We have it in design